src/main/java/com/spring/onlinebookstore/config/OpenApiConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package com.spring.onlinebookstore.config;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Contact;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.servers.Server;
+import java.util.List;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class OpenApiConfiguration {
+
+    @Bean
+    public OpenAPI defineOpenApiInfo() {
+        Server server = new Server();
+        server.setUrl("http://localhost:8080");
```

Comment:
Suggestion
don't hardcode the url, you can set it up in application properties and use `@Value` to inject